frontend/Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM node:12.2.0-stretch-slim
+FROM node:current-stretch-slim
 
-RUN mkdir -p /maisie_node_modules
+COPY . . 
 
-COPY . /app 
-WORKDIR /app 
-RUN yarn install
+# RUN /bin/bash -c mount --bind /maisie_node_modules /app/node_modules
+
+RUN yarn global add create-react-app
+RUN yarn global add react-scripts
 
-RUN /bin/bash -c mount --bind /maisie_node_modules /app/node_modules
```

Comment:
Getting rid of this command might actually break the container when the volume is bound to a filesystem hosted on Windows. I will try to check whether this statements still holds true.

Reply:
fixed by adding node_modules to .dockerignore